modules/DMZ/project_config/README.md

```diff
@@ -0,0 +1,101 @@
+# Project Configuration
+_Handles all user input and output_
+
+## Overview
+While there are a many modules & files in this project, as a _user_, I expect there to be a single point of entry where I can go and take care of everything.
+# Purpose

```

Comment:
The fact that some of these headings have an empty line before them and some don't, is somewhat vexing to me.